kombu/abstract.py

```diff
@@ -23,9 +37,9 @@ class Object:
     Supports automatic kwargs->attributes handling, and cloning.
     """
 
-    attrs = ()
+    attrs: Tuple[Tuple[str, Any], ...] = ()
```

Comment:
The thing here is that each `attr` can really be `Any`, and `str` is the name of the `attr`. It looks fine as well.